apps/skolplattformen-app/components/__tests__/Children.test.js

```diff
@@ -125,7 +125,7 @@ test('renders child in high school', () => {
 
   expect(screen.getByText('Test Testsson')).toBeTruthy()
   expect(
-    screen.getByText(translate('abbrevations.upperSecondarySchool'))
+    screen.getByText(translate('abbreviations.upperSecondarySchool'))
```

Comment:
This and similar changes will break localizations unless the corresponding changes are applied.

I could split out these changes if that helps.